src/engine/ruleset/decoders/fortinet/client-endpoint.yml

```diff
@@ -0,0 +1,56 @@
+---
+name: decoder/fortinet-client-endpoint/0
+
+metadata:
+  title: Decoder for Fortinet FortiClient Endpoint Security
+  description: Decoder for Fortinet FortiClient Endpoint Security
+  compatibility: >
+    Wazuh Engine
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/09
+  references:
+    - https://docs.fortinet.com/document/forticlient/7.0.2/log-message-reference/327109
+
+sources:
```

Comment:
Question-desing: Did you try to see if it goes through the queue-syslog decoder first? If it goes through this it is sure that it goes through the queue-localfile so I suppose so. Please add queue-syslog to the source

Reply:
Not apply for this. decoder/queue-syslog/0  ->  failure